yaksh/evaluator_tests/test_scilab_evaluation.py

```diff
@@ -71,6 +73,8 @@ def test_infinite_loop(self):
         result = get_class.evaluate(**kwargs) 
         self.assertFalse(result.get("success"))
         self.assertEqual(result.get("error"), self.timeout_msg)
+        time.sleep(4)
+        self.assertEqual(1, subprocess.call(["pgrep", "scilab-cli-bin"]))
```

Comment:
I agree with this assessment. It would be nice to have this tested differently.  A possibility is to use mock or to use psutil depending on what it is you are really intending to test.  What is the intent of this test?